bin/status.py

```diff
@@ -34,6 +36,13 @@ def main():
     parser = ArgumentParser()
     parser.add_argument("config", help="json cluster configuration file")
     args = parser.parse_args()
+    if 'config.json' not in args.config:
+        print("Please provide config.json")
+        print("Run : python3 status.py config.json")
```

Comment:
The usage message should be printed by the argument parser, it already knows what the arguments are.